### ISSUE TITLE
Lint does not complain about ignored values, returned from function call

### ISSUE THREAD

**Issue by elgris** [golang/lint#53]
Hello!

I'm just wondering if it's valid for Go code to totally ignore values returned from some func.

Example:

```
func producer() (int, int, error) {
    return 10, 20, errors.New("something")
}

func processor() {
    producer(); // MATCH /error function returns 3 values, 3 ignored/
}
```

Test case is prepared in commit https://github.com/elgris/lint/commit/6541f24d3a60caf4d0256be3a714d73806ea36ea

Cheers!
